src/system/SystemPacketBuffer.h

```diff
@@ -380,6 +380,7 @@ class DLL_EXPORT PacketBuffer : private pbuf
 #endif
 
     void AddRef();
+    uint16_t GetRef() { return this->ref; }
```

Comment:
If we do this, I'd really rather expose a boolean "do I have sole ownership?" value than the refcount in general.  The latter is not really the business of anyone outside this class; it has no useful semantic meaning to consumers other than "is it 1?"

Reply:
Acked. I use function bool IsRetained() to check is the buffer is solely owned, please let me know if there is a better name.